examples/WorldWindShim.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * This shim is used to switch between the individual files of WorldWind source and the minified library. It is
+ * not required for the use of WorldWind.
```

Comment:
Good thinking adding a comment here.

@AkeluX talked this morning and would like to see a few points called out here:
- The WorldWind shim is not required to use WorldWind. You've done this already. How about we go one step further and suggest that applications include the minified library directly?
- The WorldWind shim is used to switch how we include WorldWind. Again, you've already done this. Let's mention the reason for switching: running examples from WorldWind source in the development environment vs running examples from the minified library online.